packages/editor-ui/src/components/AskAssistant/AskAssistantChat.vue

```diff
@@ -33,9 +35,10 @@ async function onUserMessage(content: string, quickReplyType?: string) {
 					(msg) => msg.role === 'assistant' && !['text', 'event'].includes(msg.type),
 				).length
 			: null;
-	if (quickReplyType === 'all-good' || quickReplyType === 'still-stuck') {
+	if (quickReplyType !== undefined) {
```

Comment:
what about `give me another suggestion` that's feedback then? should we instead check for `isFeedback` flag backend is returning?